src/planning/strategies/create-handle-group.ts

```diff
@@ -53,19 +53,19 @@ export class CreateHandleGroup extends Strategy {
           }
         }
 
-        if (maximalGroup) {
-          return recipe => {
-            const newHandle = recipe.newHandle();
-            newHandle.fate = 'create';
-            for (const {particle, connSpec} of maximalGroup) {
-              const cloneParticle = recipe.updateToClone({particle}).particle;
-              const conn = cloneParticle.addConnectionName(connSpec.name);
-              conn.connectToHandle(newHandle);
-            }
-            return 0;
-          };
+        if (!maximalGroup) {
```

Comment:
Rearranged this to use a guard rather than nesting. Not important to the rest of the changes.